plugins/module_utils/network/sonic/config/l3_interfaces/l3_interfaces.py

```diff
@@ -164,16 +166,17 @@ def _state_overridden(self, want, have, diff):
         """
         ret_requests = list()
         commands = list()
-        interfaces_to_delete = get_diff(have, want, TEST_KEYS)
-        if interfaces_to_delete:
-            delete_interfaces_requests = self.get_delete_l3_interfaces_requests(want, have)
-            ret_requests.extend(delete_interfaces_requests)
-            commands.extend(update_states(interfaces_to_delete, "deleted"))
+        new_want = self.update_object(want)
+        get_override_interfaces = self.get_interface_object_for_overridden(have)
+        diff = get_diff(get_override_interfaces, new_want, TEST_KEYS)
```

Comment:
Coverage is needed here for the case of an "overridden" task that adds more addresses to an existing address list.